src/Extensions/AzureBlobStorage/extension.json

```
@@ -1,8 +1,8 @@
 {
-  "Id": "Microsoft.Diagnositcs.Monitoring.AzureStorage",
+  "Id": "Microsoft.Diagnostics.Monitoring.AzureStorage",
   "Version": "1.0.0",
   "Program": "AzureBlobStorage.exe",
```

```suggestion
  "Program": "dotnet-monitor-egress-azureblobstorage.exe",
```

I assume this needs to be changed to this?